app/controllers/purchases_controller.rb

```diff
@@ -13,9 +13,16 @@ def index
     @paginated_purchases = @purchases.page(params[:page])
     # Are these going to be inefficient with large datasets?
     # Using the @purchases allows drilling down instead of always starting with the total dataset
+    # Purchase quantity
     @purchases_quantity = @purchases.collect(&:total_quantity).sum
     @paginated_purchases_quantity = @paginated_purchases.collect(&:total_quantity).sum
+    # Purchase value
     @total_value_all_purchases = @purchases.sum(&:amount_spent_in_cents)
+    @paginated_purchases_value = @paginated_purchases.collect(&:amount_spent_in_cents).sum
+    # Fair Market Values
+    @total_fair_market_values = @purchases.sum(&:value_per_itemizable)
+    @paginated_fair_market_values = @purchases.collect(&:value_per_itemizable).sum
```

Comment:
For @paginated_fair_market_values,  this should refer to @paginated_purchases,  not @purchases.  Otherwise, you're just getting the grand total on both (which is the behaviour I'm seeing.)